src/modules/search/SearchModule.tsx

```diff
@@ -153,20 +153,22 @@ export default function SearchBar(props: any) {
       }}
       onSubmit={form.onSubmit((values) => {
         const query = values.query.trim();
+        var open_in = config.settings.searchNewTab ?? '_self';
```

Comment:
Please fix the eslint error:
```Unexpected var, use let or const instead.eslint[no-var](https://eslint.org/docs/rules/no-var)```

Reply:
By the way, if you don't see this error, download the Eslint Extension from the Visual Studio Code marketplace:
https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint

Reply:
I installed the extension and changed it to const. I don't see any errors now.

Reply:
Thank you, you may resolve this issue